src/main/java/core/basesyntax/service/impl/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvFileReaderImpl implements CsvFileReader {
+    @Override
+    public List<String> readCsvFileToStringList(String filePath) {
+        List<String> stringListFromCsvFile;
+        try {
+            stringListFromCsvFile = Files.readAllLines(Path.of(filePath));
+            stringListFromCsvFile.remove(0);
```

Comment:
you should do this in a method where you convert a list of lines to transactions